arangodb-net-standard/Serialization/IApiClientSerialization.cs

```diff
@@ -25,7 +35,18 @@ public interface IApiClientSerialization
         /// <param name="serializationOptions">The serialization options. When the value is null the
         /// the serialization options should be provided by the serializer, otherwise the given options should be used.</param>
         /// <returns></returns>
-        byte[] Serialize<T>(T item, ApiClientSerializationOptions serializationOptions);
+        //byte[] Serialize<T>(T item, ApiClientSerializationOptions serializationOptions);
```

Comment:
Shouldn't these old methods be removed if we are replacing them?

Reply:
Done